test/integration/default/build/rds.tf

```diff
@@ -0,0 +1,80 @@
+# Contains resources and outputs related to testing the aws_rds_* resources.
+
+#======================================================#
+#                    RDS Instances
+#======================================================#
+
+
+resource "aws_db_instance" "default" {
+  allocated_storage    = 20
+  storage_type         = "gp2"
+  engine               = "mysql"
+  engine_version       = "5.6.37"
+  instance_class       = "db.t2.micro"
+  identifier           = "test-instance-id"
+  name                 = "test_instance"
+  username             = "testuser"
+  password             = "testpassword"
+  parameter_group_name = "default.mysql5.6"
+  skip_final_snapshot  = true
+}
+
+output "rds_db_instance_address" {
```

Comment:
Why are you outputting so many things?  You're only using `rds_db_instance_id` in the integration test.